src/js/login/containers/Main.jsx

```diff
@@ -27,11 +27,11 @@ class Main extends React.Component {
 
     // TODO: Remove this conditional statement when going to production.
     if (__BUILDTYPE__ !== 'production') {
-      this.serverRequest = $.get(`${environment.API_URL}/v0/sessions/new`, result => {
+      this.serverRequest = $.get(`${environment.API_URL}/v0/sessions/new?level=1&location=/auth/login/callback/`, result => {
```

Comment:
Granted, passing it wont actually do any harm, but probably best to just remove it.